pkg/webui/components/button/button.styl

```diff
@@ -15,6 +15,8 @@
 $button($color)
   reset-button()
   one-liner()
+  display: inline-flex
```

Comment:
This line breaks text alignment for `<Button large />`
<img width="187" alt="Screenshot 2019-04-01 at 16 41 40" src="https://user-images.githubusercontent.com/16374166/55336359-1460d600-549d-11e9-9b93-9f43459113cb.png">